.github/dependabot.yml

```diff
@@ -89,7 +75,29 @@ updates:
         patterns:
           - "cypress"
           - "eslint-plugin-cypress"
-
+    ignore:
+      - dependency-name: "@types/node"
+        update-types: ["version-update:semver-major"]
+      - dependency-name: "next"
+        versions: [">=15.4.0-canary.47"]
+      - dependency-name: "@next/*"
+        versions: [">=15.5.2"]
```

Comment:
What is the relationship between `next` and `@next/*` versions? Should they be the same like it is the case for e.g. Storybook?

To me it seems to be the case since the both stem from the same monorepo at https://github.com/vercel/next.js.

If this is the case then I suggest pinning them to the same version e.g. `15.4.0` and ignoring further updates. Other `@next/*` packages are currently at `15.0.3`.